examples/angular/src/app/codat-link/codat-link.component.ts

```diff
@@ -23,22 +23,27 @@ export class CodatLinkComponent implements AfterViewInit {
       const target = this.target.nativeElement;
       const companyId = this.companyId;
 
-      import('https://link-sdk.codat.io').then(({ CodatLink }) => {
-        new CodatLink({
-          target,
-          props: {
-            companyId,
-            onClose: () => this.close.emit(),
-            onConnection: (connection) =>
-              alert(`On connection callback : ${connection.connectionId}`),
-            onError: (error) => {
-              alert(`On error callback : ${error.message}`);
-              this.close.emit();
+      // webpackIgnore is a magic comment that prevents webpack from
+      //   parsing this dynamic import. The build will fail otherwise.
+      // See https://webpack.js.org/api/module-methods/#magic-comments
+      import(/* webpackIgnore: true */ 'https://link-sdk.codat.io').then(
```

Comment:
If webpack ignore isn't needed, should it be in the example?

Reply:
We were having issues with build without it. We have applied the same fix to Next App I believe. 


https://teamcodat.slack.com/archives/C0560S3N0AW/p1689757419051459